raisinbread/RB_files/RB_files.sql

```diff
@@ -1,316 +1,314 @@
-INSERT INTO `files` (`FileID`, `SessionID`, `File`, `SeriesUID`, `EchoTime`, `CoordinateSpace`, `OutputType`, `AcquisitionProtocolID`, `FileType`, `InsertedByUserID`, `InsertTime`, `SourcePipeline`, `PipelineDate`, `SourceFileID`, `ProcessProtocolID`, `Caveat`, `TarchiveSource`, `ScannerID`, `AcqOrderPerModality`) VALUES (440,2145,'assembly/676061/V1/mri/native/demo_676061_V1_t1_001.mnc','1.3.12.2.1107.5.2.32.35008.2010081720232532169233614.0.0.0',0.00316,'native','native',44,'mnc','cecile',1558709277,NULL,NULL,NULL,NULL,0,63,5,1);
-INSERT INTO `files` (`FileID`, `SessionID`, `File`, `SeriesUID`, `EchoTime`, `CoordinateSpace`, `OutputType`, `AcquisitionProtocolID`, `FileType`, `InsertedByUserID`, `InsertTime`, `SourcePipeline`, `PipelineDate`, `SourceFileID`, `ProcessProtocolID`, `Caveat`, `TarchiveSource`, `ScannerID`, `AcqOrderPerModality`) VALUES (441,2145,'assembly/676061/V1/mri/native/demo_676061_V1_t2_001.mnc','1.3.12.2.1107.5.2.32.35008.2010081720295184960234260.0.0.0',0.497,'native','native',45,'mnc','cecile',1558709289,NULL,NULL,NULL,NULL,0,63,5,1);
-INSERT INTO `files` (`FileID`, `SessionID`, `File`, `SeriesUID`, `EchoTime`, `CoordinateSpace`, `OutputType`, `AcquisitionProtocolID`, `FileType`, `InsertedByUserID`, `InsertTime`, `SourcePipeline`, `PipelineDate`, `SourceFileID`, `ProcessProtocolID`, `Caveat`, `TarchiveSource`, `ScannerID`, `AcqOrderPerModality`) VALUES (442,2145,'assembly/676061/V1/mri/native/demo_676061_V1_dwi25_001.mnc','1.3.12.2.1107.5.2.32.35008.2010081720343224353634906.0.0.0',0.102,'native','native',66,'mnc','cecile',1558709299,NULL,NULL,NULL,NULL,1,63,5,1);
-INSERT INTO `files` (`FileID`, `SessionID`, `File`, `SeriesUID`, `EchoTime`, `CoordinateSpace`, `OutputType`, `AcquisitionProtocolID`, `FileType`, `InsertedByUserID`, `InsertTime`, `SourcePipeline`, `PipelineDate`, `SourceFileID`, `ProcessProtocolID`, `Caveat`, `TarchiveSource`, `ScannerID`, `AcqOrderPerModality`) VALUES (443,2145,'assembly/676061/V1/mri/native/demo_676061_V1_fMRI_001.mnc','1.3.12.2.1107.5.2.32.35008.2010081720415977405449710.0.0.0',0.027,'native','native',40,'mnc','cecile',1558709312,NULL,NULL,NULL,NULL,0,63,5,1);
-INSERT INTO `files` (`FileID`, `SessionID`, `File`, `SeriesUID`, `EchoTime`, `CoordinateSpace`, `OutputType`, `AcquisitionProtocolID`, `FileType`, `InsertedByUserID`, `InsertTime`, `SourcePipeline`, `PipelineDate`, `SourceFileID`, `ProcessProtocolID`, `Caveat`, `TarchiveSource`, `ScannerID`, `AcqOrderPerModality`) VALUES (444,2145,'assembly/676061/V1/mri/native/demo_676061_V1_fMRI_002.mnc','1.3.12.2.1107.5.2.32.35008.2010081720474471621362457.0.0.0',0.027,'native','native',40,'mnc','cecile',1558709324,NULL,NULL,NULL,NULL,0,63,5,2);
+SET FOREIGN_KEY_CHECKS=0;nTRUNCATE TABLE `files`;nLOCK TABLES `files` WRITE;
```

Comment:
```suggestion
SET FOREIGN_KEY_CHECKS=0;
TRUNCATE TABLE `files`;
LOCK TABLES `files` WRITE;
```

Reply:
@cmadjar You need to remove all the debug commit. It will be fixed.

Reply:
weird... Wonder where those `n` comes from...

Reply:
there are `\n` chanracters in the commands being run but those should show up as carriage returns. it looks like something happened and they instead acted as... escape characters? this might be an OS dependent issue are you on a PC ? i think windows does not recognize `\n` as carriage returns

Reply:
I'm on a mac so I am a bit surprised that it does not recognize `\n`. Go figure...